HM_1/task_2.py

```diff
@@ -0,0 +1,19 @@
+# Напишите код, который запрашивает число и сообщает является ли оно простым или составным.
+# Используйте правило для проверки: “Число является простым, если делится нацело только на единицу и на себя”.
+# Сделайте ограничение на ввод отрицательных чисел и чисел больше 100 тысяч.
+min_n = 0
+max_n = 100000
+
+result = ""
+num = int(input("Введите число: "))
+if min_n < num < max_n:
+    start = 2
+    for i in range(start, int(num ** 0.5) + 1):
+        if num % i == 0:
+            result = "Составное"
+            break
+        else: 
+            result = "Простое"   
```

Comment:
Чтобы несколько раз не менять значение переменной result можно сделать так
```suggestion
    else: 
        result = "Простое"   
```